apis/core/lib/csvw-builder/index.ts

```diff
@@ -1,17 +1,91 @@
 import * as Csvw from '@rdfine/csvw'
-import { CsvSource, Table } from '@cube-creator/model'
+import TermMap from '@rdfjs/term-map'
+import { ColumnMapping, CsvColumn, CsvMapping, CsvSource, Table } from '@cube-creator/model'
 import cf from 'clownface'
 import $rdf from 'rdf-ext'
+import RdfResource from '@tpluscode/rdfine'
+import { Initializer } from '@tpluscode/rdfine/RdfResource'
+import { cc } from '@cube-creator/core/namespace'
 import { ResourceStore } from '../ResourceStore'
+import { warning } from '../log'
+import { NamedNode, Term } from 'rdf-js'
+
+RdfResource.factory.addMixin(
+  ...Object.values(Csvw),
+)
+
+function unmappedColumn({ column }: { column: CsvColumn }): Initializer<Csvw.Column> {
+  const initializer: Initializer<Csvw.Column> = {
+    title: $rdf.literal(column.name),
+  }
+
+  initializer.suppressOutput = true
+
+  return initializer
+}
+
+function mappedColumn({ columnMapping, column, source, namespace }: { column: CsvColumn; columnMapping: ColumnMapping; source: CsvSource; namespace: NamedNode }): Initializer<Csvw.Column> {
+  return {
+    title: $rdf.literal(column.name),
+    propertyUrl: `${namespace.value}${columnMapping.targetProperty.value}`,
```

Comment:
Wouldn't this be something for the identifiers module?

Reply:
Not sure. This is about output URIs and not resources we have in our db